primitives/rpc/debug/src/proxy/v2/call_list.rs

```diff
@@ -466,9 +627,488 @@ impl ListenerT for Listener {
 			Event::Runtime(runtime_event) => self.runtime_event(runtime_event),
 			Event::Evm(evm_event) => self.evm_event(evm_event),
```

Comment:
Can we move `self.record_transaction_event_only = false;` here and remove them individually from each event variant handler? (there's none in Suicide but I think it makes no difference)

Reply:
Hmm, I think it's risky as if for some reason we have to add new EvmEvent in the future they might not be relevant for that situation and we might forget about it and then don't understand what's going wrong.

Reply:
Yep, you are right. Nvd!